rpc/gnmi/mixed-schema.md

```diff
@@ -44,14 +41,14 @@ within the message is uniquely identified by the tuple of `<origin, path>`.
 The `origin` field is valid in any context of a `Path` message. Typically it is
 used:
 
- * In a `SetRequest` to indicate a particular schema is being used to modify
-   the target configuration.
- * In a `GetRequest` to retrieve the contents of a particular schema, or in
-   a `GetResponse` to indicate that the payload contains data from a
-   particular `<origin, path>` schema.
- * In a `SubscribeRequest` to subscribe to paths within a particular schema,
-   or `SubscribeResponse` to indicate an update corresponds to a particular
-   `<origin, path>` tuple.
+* In a `SetRequest` to indicate a particular schema is being used to modify
+  the target configuration.
+* In a `GetRequest` to retrieve the contents of a particular schema, or in
+  a `GetResponse` to indicate that the payload contains data from a
+  particular `<origin, path>` schema.
+* In a `SubscribeRequest` to subscribe to paths within a particular schema,
+  or `SubscribeResponse` to indicate an update corresponds to a particular
+  `<origin, path>` tuple.
```

Comment:
I agree, I will add this clarification for the `cli` case.

Reply:
Thx @dplore - I'll leave this to another PR/issue to track but want to mention that a somewhat parallel issue is also seen that while the `Capabilities` RPC reports "supported encodings" I think we will want to clarify that this type of reporting is mostly meaningless as each RPC (at least in the context of gNMI) cannot support nor _should_ support all encodings for all operations - its similar in the fact that we have per RPC/origin/encoding behaviors coming out of this that need to be carefully documented.

Reply:
One thing to think about: this means that you must support a `GetRequest` for the origin CLI, which means no chunking of the configuration into separate gRPC messages. Is this desirable?

It seems like we might want to allow this to be something that COULD be implemented in the spec, but not require it for an initial implementation (i.e., it's a MAY).

Reply:
This imo goes back to proper definition and divide of origins and encodings to resolve this one.

`Get` is problematic from a unary RPC perspective among a few other artifacts I feel but I'll leave that for another discussion.

If we stick w/ `cli` and allow for this in `Subscribe` to circumvent the scaling cases, it would only ever be for the configuration (e.g. `show config`) - why?  Well because origin `cli` is really only focused on config - I will discount the implementation mentioned in another comment here that is leveraging `Get` for packing "show commands" but this would open up an entirely other class of expectations, overloading usage, etc.. that I think is best served via other means.

If we draw this back to an origin of `native` - `Subscribe` would already be intended to be supported for native in order to request modeled config/state (mostly PROTO encoding in reality) - if the encoding is of say the new TypedValue `cli_val` then you only receive config.  As far as the path requested, this is where this would need to be more well defined for this encoding type whether always for the full configuration or a subset by way of a negotiated PathElem "filter" (Since to date we have been mostly focused on XPath representation vs. CLI command arguments)